code/css/mobile.css

```diff
@@ -0,0 +1,69 @@
+
+.container {
+  width:100%;
+  margin: 0 auto;
+}
+
+.headercontainer {
```

Comment:
apply snake casing using a "-" between each word. It's a standard practice and make it easier to read.